tests/test_routes.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+
+# TEST GET ALL PLANETS
+def test_get_all_planets_with_no_records(client):
+    # Act
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == []
+
+# TEST ONE PLANET
+def test_get_one_planet(client, two_saved_planets):
```

Comment:
If you added a single saved planet fixture, you could use that to obtain the id instead of hard-coding it like below